### PR TITLE
Documented how to configure Syslog and DB connection pool in harbor.yml

### PR DESCRIPTION
**Pull request by stuclem**
@reasonerjt and @xaleeks does this cover it?

### CODE REVIEW

**Comment by xaleeks** [comment]
LGTM, but @ninjadq please review carefully this is your baby :)

File: docs/installation_guide.md [hunk 1]
```diff
@@ -103,16 +103,23 @@ The parameters are described below - note that at the very least, you will need
 
 
 - **database**: the configs related to local database
-  - **password**: The root password for the PostgreSQL database used for **db_auth**. _Change this password for any production use!_
+  - **password**: The root password for the PostgreSQL database used for **db_auth**. Change this password for any production use!
+  - **max_idle_conns**: The maximum number of connections in the idle connection pool. If <=0 no idle connections are retained.
+  - **max_open_conns**: The maximum number of open connections to the database. If <= 0 there is no limit on the number of open connections. The default number of connections is 100 for Postgres.
```
Comment (ninjadq) [inline docs/installation_guide.md]: the default value is 2

File: docs/installation_guide.md [hunk 2]
```diff
@@ -103,16 +103,23 @@ The parameters are described below - note that at the very least, you will need
 
 
 - **database**: the configs related to local database
-  - **password**: The root password for the PostgreSQL database used for **db_auth**. _Change this password for any production use!_
+  - **password**: The root password for the PostgreSQL database used for **db_auth**. Change this password for any production use!
+  - **max_idle_conns**: The maximum number of connections in the idle connection pool. If <=0 no idle connections are retained.
```
Comment (ninjadq) [inline docs/installation_guide.md]: the default value of `max_idle_conns` is `2`

File: docs/installation_guide.md [hunk 3]
```diff
@@ -103,16 +103,23 @@ The parameters are described below - note that at the very least, you will need
 
 
 - **database**: the configs related to local database
-  - **password**: The root password for the PostgreSQL database used for **db_auth**. _Change this password for any production use!_
+  - **password**: The root password for the PostgreSQL database used for **db_auth**. Change this password for any production use!
```
Comment (ninjadq) [inline docs/installation_guide.md]: This is not for `db_auth` only, it is for the user who uses local DB, the other auth method like ldap, oidc also need this if they did not use an external database.

**Comment by heww** [comment]
@stuclem hi, Stuart, this pr miss the max_idle_conns and max_open_conns for harbor config in external_database.

WX20190916-140527

File: docs/installation_guide.md [hunk 4]
```diff
@@ -100,19 +100,24 @@ The parameters are described below - note that at the very least, you will need
 
 - **harbor_admin_password**: The administrator's initial password. This password only takes effect for the first time Harbor launches. After that, this setting is ignored and the administrator's password should be set in the Portal. _Note that the default username/password are **admin/Harbor12345** ._
 
-
-
 - **database**: the configs related to local database
-  - **password**: The root password for the PostgreSQL database used for **db_auth**. _Change this password for any production use!_
+  - **password**: The root password for the PostgreSQL database. Change this password for any production use.
+  - **max_idle_conns**: The maximum number of connections in the idle connection pool. If <=0 no idle connections are retained. The default value is 2.
```
Comment (heww) [inline docs/installation_guide.md]: **max_idle_conns**: The maximum number of connections in the idle connection pool. If <=0 no idle connections are retained. Note that the default value is 50 and if it is not configured the value is 2.

File: docs/installation_guide.md [hunk 5]
```diff
@@ -100,19 +100,24 @@ The parameters are described below - note that at the very least, you will need
...
+  - **max_open_conns**: The maximum number of open connections to the database. If <= 0 there is no limit on the number of open connections. The default value is 0 (unlimited), except for Postgres, for which the default number of connections is 100.
```
Comment (heww) [inline docs/installation_guide.md]: **max_open_conns**: The maximum number of open connections to the database. If <= 0 there is no limit on the number of open connections. Note that the default value is 100 for the max connections of harbor db is 100 and if it is not configured the value is 0.

File: docs/installation_guide.md (hunk 1 above)
Comment (reasonerjt) [reply docs/installation_guide.md]: @stuclem Please make update according to heww's comment, sorry for the confusion, and I'm resolving this comment.

File: docs/installation_guide.md (hunk 4 above)
Comment (stuclem) [reply docs/installation_guide.md]: Fixed in https://github.com/goharbor/harbor/pull/9005/commits/0afffd027ee83f120e31ed7dd7067a91d27d89a0.

File: docs/installation_guide.md (hunk 5 above)
Comment (stuclem) [reply docs/installation_guide.md]: Fixed in https://github.com/goharbor/harbor/pull/9005/commits/55de600b2ae0a7bbfb64d9640ccf9c2413dabb47.

**Comment by stuclem** [comment]
> @stuclem hi, Stuart, this pr miss the max_idle_conns and max_open_conns for harbor config in external_database.

Fixed in https://github.com/goharbor/harbor/pull/9005/commits/54323ec208df9e24965cba993463b818aad3a095. @heww is this PR OK now? Thanks!

File: docs/installation_guide.md [hunk 6]
```diff
@@ -143,6 +148,8 @@ refer to **[Configuring Harbor with HTTPS Access](configure_https.md)**.
     - **username**: username to connect harbor core database
     - **password**: password to harbor core database
     - **ssl_mode**: is enable ssl mode
+    - **max_idle_conns**: The maximum number of connections in the idle connection pool. If <=0 no idle connections are retained. The default value is 50 and if it is not configured the value is 2.
+    - **max_open_conns**: The maximum number of open connections to the database. If <= 0 there is no limit on the number of open connections. The default value is 100 for the max connections to the Harbor database. If it is not configured the value is 0.
```
Comment (heww) [inline docs/installation_guide.md]: Sorry, for external database max_idle_conns and max_open_conns default values are 2 and 0.

Comment (stuclem) [reply docs/installation_guide.md]: Fixed in https://github.com/goharbor/harbor/pull/9005/commits/da25de33a5ec397a2f49aa3acdc2af0b440c38fb.

**Pull request merged**